README.md

```diff
@@ -24,40 +24,34 @@
 - Twitter: [@abol_ger](https://twitter.com/ger_abol)
 - LinkedIn: [David Ger](https://www.linkedin.com/in/david-ger-426b4576/)
 
-👤 **David Ger**
+👤 **Lusindiso Ntanjana**
 
-- GitHub: [@ger619](https://github.com/ger619)
-- Twitter: [@abol_ger](https://twitter.com/ger_abol)
-- LinkedIn: [David Ger](https://www.linkedin.com/in/david-ger-426b4576/)
-
-## Collaborators
-
-
-
-## 🤝 Contributing
+- GitHub: [@Lusindiso](https://github.com/Lusindiso)
+- Twitter: [@LusindisoNt](https://twitter.com/LusindisoNt)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lusindisontanjana/)
 
-Contributions, issues, and feature requests are welcome!
+## Acknowledgments
 
-## Getting Started
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
 
-To get a local copy follow these simple steps:
+## Video Link
 
-- Copy this link .
+> Here is a video link to our work [link](https://drive.google.com/file/d/15LNvD-5TcJutTbQ7SP69JQTDYTssRhOX/view?usp=sharing)
 
-- Create a local directory that you want to clone the repository.
 
-- Open the command prompt in the created directory.
+## 🤝 Contributing
 
-- On the terminal run this command git clone 
+Contributions, issues, and feature requests are welcome!
 
-- Go to the repository folder using command prompt cd javascript-capstone.
+Feel free to check the [issues page](../../issues/).
 
-- Install the dev dependencies for linters run npm install.
+## Show your support
 
-## Acknowledgments
+Give a ⭐️ if you like this project!
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+## 📝 License
 
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] This MIT link is broken. It would be better to fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.